service/models.py

```diff
@@ -1,89 +1,109 @@
 """
-Models for YourResourceModel
+Models for Customer
 
 All of the models are stored in this module
 """
 import logging
+from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 
 logger = logging.getLogger("flask.app")
 
 # Create the SQLAlchemy object to be initialized later in init_db()
 db = SQLAlchemy()
 
-
 # Function to initialize the database
 def init_db(app):
     """ Initializes the SQLAlchemy app """
-    YourResourceModel.init_db(app)
+    Customer.init_db(app)
 
 
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
 
-class YourResourceModel(db.Model):
+class Customer(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a Customer
     """
 
-    app = None
-
+    ###############
     # Table Schema
+    ##############
+
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    first_name = db.Column(db.String(255), nullable=False)
+    last_name = db.Column(db.String(255), nullable=False)
+    email = db.Column(db.String(255), nullable=False)
+    password = db.Column(db.String(255), nullable=False)
+    # Phone Number
```

Comment:
Definitely we can do this, but it might be redundant as the date of each log might be already captured when call the logging function. Therefore, we need to check the documentation of the function.

Reply:
I just check the log_handlers.py and I found the following: `formatter = logging.Formatter("[%(asctime)s] [%(levelname)s] [%(module)s] %(message)s", "%Y-%m-%d %H:%M:%S %z")`. Based on this, I think the the time and date is automatically captured by the server along other things.

Reply:
Okay, that's a good point. However, I'm not sure where these logs are being stored. Was checking the gunicorn and Flask documentation, but couldn't find the logs file path. 
I feel it'll be better to include the two fields in Customer for now to handle the REST API calls accordingly since we plan to implement Log Schema in later sprints. Can discuss more in the standup call and resolve this.

Reply:
Followed up with Prof.